### PR TITLE
Added link for Sable

### PR DESCRIPTION
**Pull request by codefruitio**

### CODE REVIEW

**Comment by tillkruss** [comment]
Let's add an "Open Sable" button to the sheets in "Activity". You'll have to check if the app is installed using a URL scheme provided by Sable.

File: Ruddarr/Ruddarr.swift [hunk 1]
```diff
@@ -63,6 +63,7 @@ struct Links {
     static let AppStore = URL(string: "itms-apps://itunes.apple.com/app/id6476240130")!
     static let TestFlight = URL(string: "https://testflight.apple.com/join/WbWNuoos")!
     static let Discord = URL(string: "https://discord.gg/UksvtDQUBA")!
+    static let Sable = URL(string: "https://apps.apple.com/us/app/sable/id6630387095")!
```
Comment (tillkruss) [inline Ruddarr/Ruddarr.swift:66]: Let's remove `us` from the link to make it universal.

File: Ruddarr/Views/Activity/ActivityView+Toolbar.swift [hunk 2]
```diff
@@ -39,6 +39,9 @@ extension ActivityView {
             }
             .id(hotfixId)
         }
+        ToolbarItem(placement: .primaryAction) {
+            toolbarOpenSable
```
Comment (tillkruss) [inline Ruddarr/Views/Activity/ActivityView+Toolbar.swift:43]: I think I'll modify this to make it a button instead.

File: Ruddarr/Views/Activity/ActivityView+Toolbar.swift [hunk 3]
```diff
@@ -96,4 +99,20 @@ extension ActivityView {
                 .imageScale(.medium)
         }
     }
+    
+    var toolbarOpenSable: some View {
+        Link("Open Sable", destination: URL(string: sable) ?? Links.Sable)
```
Comment (tillkruss) [inline Ruddarr/Views/Activity/ActivityView+Toolbar.swift:104]: We should check if the URL can be opened first (to see if the app is installed). 

We also should display the button only for sabnzbs items, not all downdloads.

**Review by tillkruss — CHANGES_REQUESTED** [review]

File: Ruddarr/Views/Activity/ActivityView+Toolbar.swift (hunk 3 above)
Comment (codefruitio) [reply Ruddarr/Views/Activity/ActivityView+Toolbar.swift]: Just made the modifications to the URL and the Open Sable link should now only show if it is installed. 

Not sure how you want to tackle the per-item scenario if it is in SABnzbd or not

Comment (tillkruss) [reply Ruddarr/Views/Activity/ActivityView+Toolbar.swift]: The payload for each queue item has download-client details, like its name/type.

Comment (codefruitio) [reply Ruddarr/Views/Activity/ActivityView+Toolbar.swift]: Got it. So in that case you want the button to be shown on the QueueItemSheet for items that are queued in SABnzbd?

Comment (tillkruss) [reply Ruddarr/Views/Activity/ActivityView+Toolbar.swift]: The most reliable field would probably be parsing `downloadId`.

>So in that case you want the button to be shown on the QueueItemSheet for items that are queued in SABnzbd?

Yep

Comment (codefruitio) [reply Ruddarr/Views/Activity/ActivityView+Toolbar.swift]: I'll work on that later this weekend. Going to a wedding today!

Comment (tillkruss) [reply Ruddarr/Views/Activity/ActivityView+Toolbar.swift]:
```json
"downloadId": "SABnzbd_nzo_lef0l_33",
"protocol": "usenet",
"downloadClient": "SABnzbd",
```

**Pull request closed without merging**